src/pypit.py

```diff
@@ -180,27 +186,33 @@ def PYPIT(redname, progname=__file__, quick=False, ncpus=1, verbose=1, logname=N
         opt, arg = getopt.getopt(sys.argv[1:], 'hqc:v:', ['help',
                                                           'quick',
                                                           'cpus',
+                                                          'use_masters',
                                                           'verbose'])
         for o, a in opt:
             if o in ('-h', '--help'):
                 initmsgs.usage(None)
-            elif o in ('-q', '--quick'):
+            elif o in ('-q', '--quick'):  # I don't think this is working right
                 qck = True
             elif o in ('-c', '--cpus'):
                 cpu = int(a)
             elif o in ('-v', '--verbose'):
                 vrb = int(a)
+        for a in arg[1:]:
```

Comment:
Like in my comment in `arload`, this line should be placed in the previous for loop (the for loop starting on line 191), with the case:

`elif o in ('m', 'use_masters'): argflag['masters']['use'] = True`